### ISSUE TITLE
Slicing predictions for using `plot_prediction()` method with Temporal Fusion Transformer

### ISSUE THREAD

**Issue by nicoliKim** [jdb78/pytorch-forecasting#1945]
Hello, 

First of all thanks a lot for putting together this nice libraries. 

I have a question regarding on how to properly call the `plot_prediction()` method. 
Assume I have a `TemporalFusionTransformer` model which I refer below as `model`. 

To make predictions I do the following:

```python
preds = model.predict(
            test_dataloader,
            mode='raw', 
            return_x=True,
            return_y=True, 
            mode_kwargs={"n_samples": n_samples}, 
            trainer_kwargs=dict(accelerator="mps")
            )
```

Now using these predictions, I want to plot the results. 
Note that if I do

```python
type(self.predictions.output.prediction)
```

this is a list of size `3`where each element of the list is a different variable I am forecasting. If I analyze the shape of the first element of the list I get

```python
self.predictions.output.prediction[0].shape
```
I get 
```python
torch.Size([1, 60, 3])
```
where the first dimension is the batch, second dimension is the prediction length and last dimension is the quantiles I obtain from the `QuantileLoss` loss function. 

What I am struggling with is the following. I'd like to call the `plot_prediction()` method so that only one element of the list `self.predictions.output.prediction` is being used. 

I hope the way I phrased the question makes sense. Please bear with me as I am new to the library and its workflow. If you have suggestions on how to do things differently that would be very much appreciated.

**Comment by fkiraly** [comment]
I think this issue should be in `pytorch-forecasting`, moving it over there.

**Comment by phoeenniixx** [comment]
Hello @nicoliKim! I don't think right now there is any way to plot only one element of the raw predictions using `plot_prediction()`, you'll have to do it maunally.

But We are working on the rework of this package for the v2.0 release and we would keep this thing in mind for the new version!
You can also look at the beta version [here](https://github.com/sktime/pytorch-forecasting/blob/main/docs/source/tutorials/ptf_V2_example.ipynb)

And if you have any feedback or suggestions for v2.0 please comment on #1736 :)

FYI @PranavBhatP, @fkiraly, @agobbifbk

**Comment by nicoliKim** [comment]
Thanks a lot @phoeenniixx for letting me know! Would be cool to see this being implemented. 
As an alternative, let's assume I have 3 variables to predict: do you think that training 3 independent models (for each of these variables) is an option for then using the `plot_predict()` method at the current stage?

Do you expect that training one model to forecast three variables (i.e., setting `target=['Var1', 'Var2', 'Var3']`) compared to having three models to forecast a single variable may lead to significant differences in performance?

**Comment by fkiraly** [comment]
> Do you expect that training one model to forecast three variables (i.e., setting target=['Var1', 'Var2', 'Var3']) compared to having three models to forecast a single variable may lead to significant differences in performance?

In general, it may perform differently if you do this. Probably worse since you are missing out on associations between the variables? Hard to say without trying though.
